quickstart/101-virtual-network-manager-create-mesh/providers.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">=1.0"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>3.56.0"
```

Comment:
`azurerm_network_manager_deployment` is added into the provider in `3.56.0`, could we use `>= 3.56.0, < 4.0` instead? The meaning of `~>` can be checked [here](https://developer.hashicorp.com/terraform/language/expressions/version-constraints#-3).

Reply:
updated